ci/Jenkinsfiles/build.groovy

```diff
@@ -88,23 +88,49 @@ pipeline {
         }
       }
     }
-    stage('Compile') {
-      steps {
-        container('maven') {
-          nxWithGitHubStatus(context: 'compile') {
-            echo """
-            ----------------------------------------
-            Compile
-            ----------------------------------------"""
-            echo "MAVEN_OPTS=$MAVEN_OPTS"
-            sh 'mvn -B -nsu -T4C install -DskipTests'
+    stage('Build') {
+      parallel {
+        stage('Compile') {
+          steps {
+            container('maven') {
+              nxWithGitHubStatus(context: 'compile') {
+                echo """
+                ----------------------------------------
+                Compile
+                ----------------------------------------"""
+                echo "MAVEN_OPTS=$MAVEN_OPTS"
+                sh 'mvn -B -nsu -T4C install -DskipTests'
+              }
+            }
+          }
+          post {
+            success {
+              archiveArtifacts artifacts: '**/target/*.jar, **/target/nuxeo-*-package-*.zip'
+              junit testResults: '**/target/surefire-reports/*.xml, **/target/failsafe-reports/*.xml', allowEmptyResults: true
```

Comment:
Not related to current change, but I think the junit step can be removed here as we are skipping the tests.